webapi/src/main/scala/org/knora/webapi/messages/admin/responder/valueObjects/GroupsValueObjectsADM.scala

```diff
@@ -22,8 +23,12 @@ object GroupIRI { self =>
     if (value.isEmpty) {
       Validation.fail(BadRequestException(GROUP_IRI_MISSING_ERROR))
     } else {
+      val isUUID: Boolean = sf.couldBeUuid(value.split("/").last)
```

Comment:
I find the function name `couldBeUuid()` confusing. Either it is or it's not, but *could* seems weird in this context

Reply:
You're partially right, because it only checks the length of string, so there is possibility checked string isn't the UUID, but I think to rename it.